web/src/SidebarResources.tsx

```diff
@@ -39,6 +49,69 @@ const NoMatchesFound = styled.li`
   color: ${Color.grayLightest};
 `
 
+const SidebarLabelSection = styled(Accordion)`
+  &.MuiPaper-root {
+    background-color: unset;
+  }
+
+  &.MuiAccordion-root,
+  &.MuiAccordion-root.Mui-expanded {
+    margin: ${SizeUnit(1 / 3)} 0;
+  }
+`
+
+const SummaryIcon = styled(CaretSvg)`
+  padding: ${SizeUnit(1 / 4)};
+  transition: all 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; /* Copied from MUI accordion */
```

Comment:
i dunno that i trust "transition: all", seems too easy to accidentally transition a property you don't expect

Reply:
fixed 👍🏻